services/src/indexer.rs

```diff
@@ -107,6 +107,9 @@ pub trait Indexer: Sized {
     /// Indexer config struct
     type Config;
 
+    /// generate a default test config
```

Comment:
please can you keep with the consistency of the crate and start each comment with a capital and make sure spacing is correct